meta-facebook/meta-python3/recipes-connectivity/bind/bind_%.bbappend

```diff
@@ -1,3 +0,0 @@
-# Python3 only. Override py2 packages
-RDEPENDS_${PN}_remove = "python-core"
-RDEPENDS_${PN} += " python3-core"
```

Comment:
Is that true for rocko as well?

Reply:
Yes, I do believe so.
https://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-connectivity/bind/bind_9.10.5-P3.bb?h=rocko